addons/product/models/product.py

```diff
@@ -19,7 +19,7 @@ class ProductCategory(models.Model):
     _rec_name = 'complete_name'
     _order = 'complete_name'
 
-    name = fields.Char('Name', index=True, required=True)
+    name = fields.Char('Name', index=True, required=True, translate=True)
```

Comment:
We can't do this. It has been revert here 322618a69c8870835e0e4ea785b7268a16d14f67. The reason being that a compute store depends on it. More info here #22060